migration/i9/postgresql/202210131724_IDEMPIERE-5450.sql

```diff
@@ -0,0 +1,7 @@
+-- IDEMPIERE-5450 Form SQL Process has security issues
+SELECT register_migration_script('202210131724_IDEMPIERE-5450.sql') FROM dual;
+
+-- Oct 13, 2022, 5:24:11 PM CEST
+INSERT INTO AD_SysConfig (AD_SysConfig_ID,AD_Client_ID,AD_Org_ID,Created,Updated,CreatedBy,UpdatedBy,IsActive,Name,Value,Description,EntityType,ConfigurationLevel,AD_SysConfig_UU) VALUES (200209,0,0,TO_TIMESTAMP('2022-10-13 17:24:11','YYYY-MM-DD HH24:MI:SS'),TO_TIMESTAMP('2022-10-13 17:24:11','YYYY-MM-DD HH24:MI:SS'),100,100,'Y','ALLOWED_KEYWORDS_IN_SQL_FORM','ALTER ,ANALYZE,COMMENT ,CREATE ,DELETE ,DROP ,GRANT ,INSERT ,REINDEX ,REVOKE ,SET ,UPDATE ,TRUNCATE ,VACUUM ','Comma separated list of commands allowed in the SQL Process form (the spaces after the keyword are relevant)','D','S','52f9e30d-4a15-4a63-890c-c338e9f20f61')
```

Comment:
The single space after keyword requirement is a rather strange one. Isn't it always a full trim of the token and append a single space after that (i.e trim(token) + " ") ?

Reply:
Well, I added that because of the command ANALYZE which doesn't require a space at the end, but maybe we can make it like an exception in code, that would be easier to understand (or the user must know that he needs to write "ANALYZE " instead of just "ANALYZE")

Reply:
That's certainly a special case, it is still "ANALYZE " if you want to analyze 1 table instead of the whole db.Anyway, some of this like vacuum, analyze, truncate and drop doesn't really sounds like something that should be run from an iDempiere form ...

Reply:
Thanks @hengsin - I changed the code to remove that requirement of the space at the end of the SysConfig, tested and now is working fine.